### PR TITLE
Vizor2 teaser page

### PR DESCRIPTION
**Pull request by joukosaastamoinen**
First version of Vizor 2 teaser site. Served from `/vizor2teaser` for now.

Test deployment here: https://jouko.vizor.lol/vizor2teaser

Things still missing:

* Mobile nav menu
* ~~Proper <title> and meta tags~~
* Browser testing and some finishing touches

Tested in latest Chrome, Safari and Firefox on Mac.

image

### CODE REVIEW

File: views/vizor2Teaser.handlebars [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {{{css 'vizor2teaser'}}}
```
Comment (gmarinov) [inline views/vizor2Teaser.handlebars]: Please grab some more `<meta>`-s from layouts/main.handlebars, specifically description, keywords, opengraph, origin trial token, favicons.

File: views/vizor2Teaser.handlebars [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    <title>Vizor</title>
+</head>
+<body>
+    <section class="hero section">
+        <script src="//360.vizor.io/scripts/embed.js" data-vizorurl="//360.vizor.io/embed/matt/arctic?autoplay=true&noheader=true" ></script>
+        <div class="hero__content section__wrapper">
+            <svg class="hero__logo" width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (gmarinov) [inline views/vizor2Teaser.handlebars]: have sent you an svgo-d glyph

File: views/vizor2Teaser.handlebars [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                        </g>
+                    </g>
+                </g>
+            </svg>
+            <h1 class="hero__title h1">Build the Immersive Web with Vizor</h1>
+            <p class="hero__body">
+                Create 360 tours, stories, sites and WebVR experiences as easy as drag and drop.
+            </p>
+            <button class="hero__scroll-down scroll-down-button">
+                <svg class="scroll-down-button__icon" width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (gmarinov) [inline views/vizor2Teaser.handlebars]: svgo. on more complex designs, best to `<use>` from a sprite sheet.

File: views/vizor2Teaser.handlebars [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_18aabb3ca7b701a5acacc8aca_97a6941be8" tabindex="-1" value=""></div>
+                <button class="register__submit button">Register</button>
+            </div>
+        </div>
+    </section>
+
+    <footer class="footer section">
+        <div class="section__wrapper footer__wrapper">
+            <ul class="footer__link-list">
+                <li class="footer__link-list-item"><a href="https://vizorvr.herokuapp.com/" target="_blank">Public Slack</a></li>
```
Comment (gmarinov) [inline views/vizor2Teaser.handlebars]: `footer__link-list-item` is redundant

Comment (joukosaastamoinen) [inline views/vizor2Teaser.handlebars]: It is references in the styles: https://github.com/vizorvr/vizor/pull/1994/files#diff-193219a823aec21c2bec8feccf3b5f3eR383

File: less/build_vizor2teaser.less [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+        padding-top: 150px;
+        padding-bottom: 150px;
+    }
+    .footer__link-list {
+        display: flex;
+        flex-wrap: wrap;
+        margin-left: -15px;
+        line-height: inherit;
+    }
+    .footer__link-list-item {
```
Comment (gmarinov) [inline less/build_vizor2teaser.less]: why not nest as `li { ... }`inside the .footer__link-list?

File: less/build_vizor2teaser.less [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+}
+.footer__info {
+    margin-top: 60px;
+    strong {
+        font-weight: bold;
+    }
+    a {
+        text-decoration: underline;
+    }
+    a:hover {
```
Comment (gmarinov) [inline less/build_vizor2teaser.less]: you can nest these as `&:hover { ... }`, `&:active { ... }` etc inside the `a { }`.

File: less/build_vizor2teaser.less (hunk 5 above)
Comment (joukosaastamoinen) [inline less/build_vizor2teaser.less]: Works that way too. A long time ago, I adopted a habit of (almost) never styling HTML tags directly.

Comment (gmarinov) [inline less/build_vizor2teaser.less]: easier on the brain, and keeps the css namespace clean

**Comment by gmarinov** [comment]
happy to get a mobile version in a separate PR

**Comment by gmarinov** [comment]
screen shot 2017-04-11 at 12 54 16

hero container could use a `user-select: none` there

**Comment by joukosaastamoinen** [comment]
@gmarinov I didn't quite catch that. The intended behavior is that user can select the text, but clicking anywhere outside the text will pan the camera.

Do you think that's wrong?

**Comment by gmarinov** [comment]
@joukosaastamoinen 

hero-drag

**Comment by kschzt** [comment]
looks good to me - we just have to make sure both vizor.io and patches.vizor.io work off this same code. see https://github.com/vizorvr/vizor/blob/develop/app.js#L268

**Pull request merged**